modules/auxiliary/scanner/misc/sunrpc_portmapper.rb

```diff
@@ -22,18 +22,22 @@ def initialize
         ],
       'License'	=> MSF_LICENSE
     )
+    register_options([
+      OptEnum.new('PROTOCOL', [true, 'Protocol to use', 'tcp', %w{tcp udp}]),
```

Comment:
I'd use the `%w[]` [syntax](https://github.com/rubocop-hq/ruby-style-guide#percent-literal-braces) here and in the NIS modules. I hadn't changed my style yet.